frontend_tests/zjsunit/zjquery.js

```diff
@@ -156,6 +156,10 @@ exports.make_zjquery = function () {
                 return self.wrapper;
             },
             select: function (arg) {
+                if (arg === undefined &&
```

Comment:
Yago and I had a long discussion about the case when you call `select()` in real code that essentially does nothing.  We came to the conclusion that you can just bind something to noop in your tests.  I can explain this over chat.